package.json

```diff
@@ -65,7 +65,7 @@
   },
   "scripts": {
     "prepublishOnly": "npm run build",
-    "test": "node node_modules/karma/bin/karma start --single-run",
+    "test": "node node_modules/karma/bin/karma start --single-run && tsc --noEmit",
```

Comment:
Adding typescript compilation to `npm test` so that people don't inadvertently break things that might not be covered by unit tests.